iothub/device/tests/DeviceClientTests.cs

```diff
@@ -288,13 +289,15 @@ public async Task DeviceClient_OnMethodCalled_MethodRequestHasEmptyBody_With_Set
             deviceClient.InnerHandler = innerHandler;
 
             bool isMethodHandlerCalled = false;
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Can you explain what is this for?

Reply:
We're calling a method that has an Obsolete attribute. Calling that produces a warning to let the caller know. We are testing the methods still, but I don't want a warning in our build. :)